wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/ShareableDebugReportView.swift

```diff
@@ -0,0 +1,127 @@
+        return documentIconView
+    }()
+
+    // MARK: - Life cycle
+
+    convenience init() {
+        self.init(frame: .zero)
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+        createConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Setup
+
+    private func setupViews() {
+        addSubview(labelsView)
+        addSubview(documentIconView)
+        labelsView.addSubview(topLabel)
+        labelsView.addSubview(bottomLabel)
+    }
+
+    private func createConstraints() {
+        [
+            labelsView,
+            topLabel,
+            bottomLabel,
+            documentIconView
+        ].forEach { $0.translatesAutoresizingMaskIntoConstraints = false }
+
+        NSLayoutConstraint.activate([
+            topLabel.topAnchor.constraint(equalTo: labelsView.topAnchor),
+            topLabel.leftAnchor.constraint(equalTo: labelsView.leftAnchor),
+            topLabel.rightAnchor.constraint(equalTo: labelsView.rightAnchor),
+
+            bottomLabel.topAnchor.constraint(equalTo: topLabel.bottomAnchor, constant: 2),
+            bottomLabel.leftAnchor.constraint(equalTo: labelsView.leftAnchor),
+            bottomLabel.rightAnchor.constraint(equalTo: labelsView.rightAnchor),
+            bottomLabel.bottomAnchor.constraint(equalTo: labelsView.bottomAnchor),
+
+            labelsView.leftAnchor.constraint(equalTo: documentIconView.rightAnchor, constant: 12),
+            labelsView.rightAnchor.constraint(equalTo: rightAnchor, constant: -12),
+            labelsView.centerYAnchor.constraint(equalTo: centerYAnchor),
+
+            documentIconView.topAnchor.constraint(equalTo: topAnchor, constant: 12),
+            documentIconView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -12),
+            documentIconView.widthAnchor.constraint(equalToConstant: 32),
+            documentIconView.heightAnchor.constraint(equalToConstant: 32),
+            documentIconView.centerYAnchor.constraint(equalTo: centerYAnchor),
+            documentIconView.leftAnchor.constraint(equalTo: leftAnchor, constant: 12)
+        ])
+    }
+
+    // MARK: - Interface
+
+    func configure(with fileMetadata: ZMFileMetadata) {
+
+        let ext = (fileMetadata.filename as NSString).pathExtension
+        let dot = String.MessageToolbox.middleDot
+        let fileSize = ByteCountFormatter.string(
+            fromByteCount: Int64(fileMetadata.size),
+            countStyle: .binary
+        )
+
+        let firstLine = fileMetadata.filename.uppercased()
+        let secondLine = "\(fileSize) \(dot) \(ext)".uppercased()
+
+        topLabel.attributedText = firstLine && UIFont.smallSemiboldFont && SemanticColors.Label.textDefault
+        bottomLabel.attributedText = secondLine && UIFont.smallLightFont && SemanticColors.Label.textCollectionSecondary
```

Comment:
We want to move away from using Fontspec. I would recommend using UIFont.font(for: `WireTextStyle`)

WireTextStyle = Text styles defined in Wire's design system.